src/data_object.js

```diff
@@ -101,6 +102,22 @@ window.DataObject = class DataObject {
                     tmp_row[col] = cell;
                   }
                   break;
+                case "Date":
+                  if (normalize_date) {
+                    var d = new Date(cell),
```

Comment:
Some date format detection and shaping is necessary before this will work, `cell` can be in a lot of formats that wont work with `new Date()`.